### ISSUE TITLE
Tab outputs a newline character when the input is empty

### ISSUE THREAD

**Issue by michelschinz** [ivan-tkatchev/tab#4]
Hi,

In the tutorial, it is said that `tab '@'` is equivalent to `cat`, but it isn't so as `tab` will output a newline character when the input is empty. Compare:
```
$ < /dev/null cat|hexdump -C
```
with:
```
$ < /dev/null tab '@'|hexdump -C
00000000  0a                                                |.|
00000001
```

(Edit: in fact, `tab` adds a newline each time the input does not end with one, not only when the input is empty. In any case, this behavior can be annoying.)

Thanks for `tab`, which I really enjoy.

**Comment by ivan-tkatchev** [comment]
Yes, currently all values are terminated by a newline, and an empty sequence is still a value.
Changing this might be difficult. Maybe easier to change the documentation. :)
Is this an issue that causes problems in production?

**Comment by michelschinz** [comment]
I don't use `tab` in production at the moment, so no worries :-).

That said, I did encounter the issue for real, yes, and I ended up using `ripgrep` instead of `tab` just because of that (it happens that `ripgrep`'s `--replace` option was sufficient in that case).

Here is what I was trying to do: I had to run a query on a lot of files, and prefix each line of the result with a string that depended on the name of the file. I did that by using GNU `parallel`, and for each file I searched the information and piped the output to tab, to add the prefix. And the problem is that for ~1500 files, the query didn't return anything, which caused tab to add ~1500 empty lines to the result.

I understand that it's probably not easy to fix, but as you're probably aware, this makes `tab` behave differently than pretty much all the other similar tools around. (I just checked `ripgrep`, `grep`, `sed` and `awk`, but I'm pretty sure it's the universal behavior).
